packages/path_provider/example/pubspec.yaml

```diff
@@ -18,7 +18,9 @@ dev_dependencies:
     sdk: flutter
   integration_test_tizen:
     path: ../../integration_test/
-  permission_handler: ^6.1.1
+  permission_handler: ^8.1.3
+  permission_handler_tizen:
+    path: ../../permission_handler
```

Comment:
The example app will fail to run on TV devices if this dependency is added by default.

```
E/ConsoleMessage( 5302): FlutterApplication.cs: Run(83) > Unhandled exception: System.DllNotFoundException: Unable to load shared library 'flutter_plugins.so' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: libflutter_plugins.so: cannot open shared object file: No such file or directory
E/ConsoleMessage( 5302):    at Runner.GeneratedPluginRegistrant.IntegrationTestPluginRegisterWithRegistrar(FlutterDesktopPluginRegistrar registrar)
E/ConsoleMessage( 5302):    at Runner.GeneratedPluginRegistrant.RegisterPlugins(IPluginRegistry registry) in /home/swift/Git/plugins/packages/path_provider/example/tizen/flutter/GeneratedPluginRegistrant.cs:line 22
E/ConsoleMessage( 5302):    at Runner.App.OnCreate() in /home/swift/Git/plugins/packages/path_provider/example/tizen/App.cs:line 11
E/ConsoleMessage( 5302):    at Tizen.Applications.CoreBackend.UICoreBackend.OnCreateNative(IntPtr data)
E/ConsoleMessage( 5302):    at Tizen.Applications.CoreBackend.UICoreBackend.Run(String[] args)
E/ConsoleMessage( 5302):    at Tizen.Applications.CoreApplication.Run(String[] args)
```

Reply:
Looks very familiar...I'll will look into this.

Reply:
Wasn't it unclear for you? TV apps shouldn't depend on permission_handler_tizen because libprivacy_privilege_manager.so is not available on them.

Reply:
Oh, I got it. Overlooked the limitation part.
Since the example tests `path_provider`, I'll skip the test cases that require `permission_handler` and remove its dependencies.